R/dots.R

```diff
@@ -97,7 +97,8 @@ dots_clean_empty <- function(dots, is_empty, ignore_empty) {
 
   if (n_dots) {
     which <- match.arg(ignore_empty, c("trailing", "none", "all"))
-    switch(which,
+    switch(
+      which,
```

Comment:
We could but it would involve creating a `text_before` column in our parse data and it is not covered by the tidyverse style guide. In addition, I was not sure which of the below versions would be correct:

```r
switch(cond,
  x, 
  y
)

```

or 
```r
switch(cond,
       x, 
       y
)

```